services/net/content/ContentManager.cs

```diff
@@ -276,13 +276,12 @@ private async Task HandleMessageAsync(ConsumeResult<string, SourceContent> resul
     /// <summary>
     /// Send message to kafka with updated transcription.
     /// </summary>
-    /// <param name="sourceContent"></param>
     /// <param name="content"></param>
     /// <returns></returns>
     /// <exception cref="InvalidOperationException"></exception>
-    private async Task<DeliveryResult<string, SourceContent>> SendMessageAsync(SourceContent sourceContent, ContentModel content)
+    private async Task<DeliveryResult<string, TranscriptRequest>> SendMessageAsync(ContentModel content)
```

Comment:
We don't need to send the content to request a transcript.  We only need to sent a request.